### DISCUSSION TITLE
[Feature] Date filter on webapp main page

### DISCUSSION THREAD

**Discussion by panoskpv** [immich-app/immich#6835] [Feature Request]
### The feature

I think it would be good to have a date filter (from {{date}} to {{date}}) in the main page for a easier album creation. 

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by Sums123** [comment]
I think this is coming shortly
![image](https://github.com/immich-app/immich/assets/29928499/dc3299cb-f355-4d14-881f-9b73ccfc330f)
https://github.com/immich-app/immich/discussions/6808
